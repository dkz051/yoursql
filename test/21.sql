CREATE DATABASE OOP;
USE OOP;
CREATE TABLE qwe(id INT, aint INT, bchar CHAR, cdouble DOUBLE, PRIMARY KEY(id));

INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 1,      11, 'c',     100.0);
INSERT INTO qwe(id, aint, bchar         ) VALUES( 2, 4144958, 'b'           );
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 3,      11, 'c',     100.0);
INSERT INTO qwe(id, aint,        cdouble) VALUES( 4,      14,      4144958.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 5,      11, 'a',     100.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 6, 4144958, 'b', 4144958.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 7,      17, 'c',     100.0);
INSERT INTO qwe(id,       bchar, cdouble) VALUES( 8,          'b', 4144958.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES( 9,      17, 'a',     900.0);
INSERT INTO qwe(id, aint, bchar, cdouble) VALUES(10,      20, 'c', 4144958.0);

SELECT * FROM qwe ORDER BY aint DESC;
SELECT * FROM qwe ORDER BY aint DESC, cdouble, bchar ASC;

DROP DATABASE OOP;
