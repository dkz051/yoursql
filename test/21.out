id	aint	bchar	cdouble
2	4144958	b	NULL
6	4144958	b	4144958
10	20	c	4144958
7	17	c	100
9	17	a	900
4	14	NULL	4144958
1	11	c	100
3	11	c	100
5	11	a	100
8	NULL	b	4144958
id	aint	bchar	cdouble
2	4144958	b	NULL
6	4144958	b	4144958
10	20	c	4144958
7	17	c	100
9	17	a	900
4	14	NULL	4144958
5	11	a	100
1	11	c	100
3	11	c	100
8	NULL	b	4144958
